### PR TITLE
Feature/hit the api

### PR DESCRIPTION
**Pull request by Braingenius2**
# Added API functionality to the Leaderboard app
In this pull request, I mostly used JavaScript to implement the following,

- [x] Create a new game using the Leaderboard  API.
- [x] Implement the "Refresh" button that will get all scores from the API (receiving data from the API and parsing the JSON).
- [x] Implement the form "Submit" button that will save a score (sending data to the API).
- [x] Use arrow functions instead of the function keyword.
- [x] Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,95 @@
...
+// Create a new game
+// const createGame = async (name) => {
+//   const url = baseURL + 'games/';
+//   const data = { name };
+
+//   const result = await sendRequest(url, 'POST', data);
+//   return result;
+// };
+
+// createGame('FGames');
```
Comment (Kamba56) [inline src/index.js]: - [ ] Kindly remove all commented pieces of code from your documents. This will make your documents messy.

File: src/index.js
Comment (Kamba56) [inline src/index.js:1]: - [Optional] Consider breaking js file into modules. This is ES6 best practice.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Braingenius2, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great work consuming the API👏👏👏
- [x] Good job using the fetch API

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — APPROVED** [review]
Hi @Braingenius2 , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You have a descriptive pull request summary
🚀 You implemented the required changes required by the previous reviewer
🚀 You have good commit messages
🚀 You used Gitflow properly
🚀 You have no linter errors


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**